NitroxServer/ConsoleCommands/KickCommand.cs

```
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using NitroxServer.ConsoleCommands.Abstract;
+using NitroxServer.GameLogic;
+using NitroxModel.Packets;
+using NitroxModel.Logger;
+
+namespace NitroxServer.ConsoleCommands
+{
+    class KickCommand : Command
+    {
+        private PlayerManager playerManager;
+
+        public KickCommand(PlayerManager playerManager)
+        {
+            Name = "kick";
+            this.playerManager = playerManager;
+        }
+
+        public override void RunCommand(string[] args)
+        {
+            try
+            {
+                Player player = playerManager.GetPlayers().Where(t => t.Name == args[0]).Single();
+                args = args.Skip(1).ToArray();
+                player.SendPacket(new UserKicked("You were kicked from the server! \n Reason: " + string.Join(" ", args)));
+            }
+            catch
```

Empty catch?  We need to do something like log an error.